users/models.py

```diff
@@ -54,13 +54,15 @@ def create_superuser(self, email, password, **extra_fields):
 
 class BusinessProfile(models.Model):
     company_name = models.CharField(blank=True, max_length=255)
+    company_website = models.CharField(blank=True, max_length=255)
+    company_phone = models.CharField(blank=True, max_length=255)
+    zipcode = models.CharField(blank=True, max_length=10)
     industry_category = models.CharField(blank=True, max_length=255)
     industry_segment = models.CharField(blank=True, max_length=255)
     experience_level = models.CharField(blank=True, max_length=255)
-    recent_project = models.CharField(blank=True, max_length=255)
-    work_seeking = models.CharField(blank=True, max_length=255)
-    summary = models.CharField(blank=True, max_length=255)
-    zipcode = models.CharField(blank=True, max_length=10)
+    recent_project = models.CharField(blank=True)
```

Comment:
@jackson- Removed the character limit because these are text areas not text fields, but I'm not sure if `CharField` should be updated too, is there something different for longer text?